open-metadata-resources/open-metadata-labs/administration-labs/understanding-digital-services.ipynb

```diff
@@ -0,0 +1,126 @@
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "ODPi Egeria is an open source project that provides open standards and implementation libraries to connect tools,\n",
+    "catalogues and platforms together so they can share information about data and technology (called metadata).\n",
+    "\n",
+    "This lab will guide you through using Egeria to manage the lifecycle of Egeria Digital Services."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The scenario\n",
+    "\n",
+    "Cocco Pharmacutical Corporate Governance wish to ensure that the new personalised services in planning can be \n",
+    "shown to meet local regulatory frameworks such as GDPR and HIPPA. Including meeting any regulatory frameworks \n",
+    "relating to security breaches relating to compromised personal data.\n",
+    "\n",
+    "Erin Overview has be tasked with an investigation into integrating Egeria's Digital Service Capabilities into\n",
+    "the Software Development Lifecyce for the new personalised services in planning.\n",
+    "\n",
+    "![Erin Overview](https://raw.githubusercontent.com/odpi/data-governance/master/docs/coco-pharmaceuticals/personas/erin-overview.png)\n",
+    "\n",
+    "In this hands-on lab Erin is going create and manage an Egeria Digital Service from concept through the different stages of \n",
+    "development, test, on-boarding, deployment through to dealing with a security incident. \n",
+    "\n",
+    "As Coco Pharmacuticals are planning a new digital service that will allow customers to purchase medicines tailored to their specific needs, the stakeholders agreed that this was seen as an opportunity to trial the Egeria Digital Service capabilities.\n",
+    "\n",
+    "The first step is to check our environment is up and running using the following command.\n",
+    "As the lifecycle of a Digital Service will require data to be accessed by all the different departments we will utilize the full Lab Environment.\n",
+    "\n",
+    "In the [Egeria Server Configuration](../egeria-server-config.ipynb) lab, Gary configured servers for the Open Metadata and Governance (OMAG) Server Platforms shown in Figure 1:\n",
+    "\n",
+    "![Figure 1](../images/coco-pharmaceuticals-systems-omag-server-platforms.png)\n",
+    "> **Figure 1:** Coco Pharmaceuticals' OMAG Server Platforms"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Checking OMAG Server Platform availability...\n",
+      "Exception: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /open-metadata/platform-services/users/garygeeke/server-platform/origin (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5cec0ffc50>: Failed to establish a new connection: [Errno 111] Connection refused',))\n",
```

Comment:
ports will now be 9443, 9444, 9445 - and must include https as the protocol

Reply:
I noticed a change in c22e3dfcf2b03fecfa9544e33258987eb91d4359 -- which seems to remove all content. Intentional? (and I realised some of the original file contained output - easy to miss)

Reply:
All clean as no lab now - can come later. closing